distributed/scheduler.py

```diff
@@ -7709,7 +7705,7 @@ def profile_to_figure(state):
         # Task stream
         task_stream = self.get_task_stream(start=start)
         total_tasks = len(task_stream)
-        timespent = defaultdict(int)
+        timespent: "defaultdict[int, float]" = defaultdict(float)
         for d in task_stream:
             for x in d.get("startstops", []):
```

Comment:
```suggestion
            for x in d["startstops"]:
```
if this ever hit default it would crash on the following line